virtualization/windowscontainers/docker/configure_docker_daemon.md

```diff
@@ -31,19 +31,19 @@ Download the Docker Engine
 
 The latest version may always be found at https://master.dockerproject.org . This sample uses the latest from the v1.13-development branch. 
 
-```none
+```powershell
 Invoke-WebRequest "https://download.docker.com/components/engine/windows-server/cs-1.12/docker.zip" -OutFile "$env:TEMP\docker.zip" -UseBasicParsing
```

Comment:
I need to look at this a bit closer for consistency. The Windows 10 documents are supposed to use v1.13-dev until v1.13 is final. Windows Server 2016 is supposed to use cs-1.12.2 which is a preview of the commercially supported Docker Engine.